src/utils/time.ts

```diff
@@ -92,3 +92,16 @@ export const sortedByDate = (
     },
   )
 }
+
+export const sortedByDateRemix = (
```

Comment:
While I do sometimes enjoy remixes I think it might be better to call this method something like `sortSingleListByDate` or `sortListByDate`